packages/manager/modules/support/src/tickets/new-ticket/new-ticket.controller.js

```diff
@@ -68,4 +71,13 @@ export default class SupportNewController {
       this.issue = null;
     }
   }
+
+  handleBackButton() {
+    if (this.step !== 'creation') {
+      return this.$state.go('support.tickets');
+    }
+
+    this.step = 'issues';
```

Comment:
Could it be possible and would it make sense to handle it as states instead ?

Reply:
This wasn't done by me, and I don't have time redo entirely the component unfortunately, but yes, this should be done and it's actually in my todo list for v2 (which is already on the way 😉 )